### DISCUSSION TITLE
Migrating to V-10 - old hooks

### DISCUSSION THREAD

**Discussion by Vladislav747** [xyflow/xyflow#2127] [General]
Hello! I have migrated to v-10 but i haven't explanation https://reactflow.dev/docs/guides/migrate-to-v10/ what to use instead of  useStoreActions, useStoreState hooks ? Maybe you should add extra info to that guide.

**Comment by moklick** [comment]
Hey @Vladislav747 

good point. We will add it. You can find the new `useStore` docs here: https://reactflow.dev/docs/api/hooks/use-store/ . We don't distinguish between `useStoreActions` and  `useStoreState` anymore and use Zustand instead of Redux.
